backend/app/controllers/api/v1/favourite_projects_controller.rb

```diff
@@ -0,0 +1,24 @@
+module API
+  module V1
+    class FavouriteProjectsController < BaseController
+      before_action :authenticate_user!
```

Comment:
We need a story to review authorization in the backend @agnessa. This endpoint and many others, should be only for users that belong to an approved account. I see that `review_status` is for project_developer and investor, not sure why that is not for account. Maybe that will be a chance to implement this with either pundit or cancancan, or if not needed then just simpler.

Reply:
I would personally go for cancancan for this kind of restrictions, but we can also pick up pundit or wrap appropriate controller action with some custom method -- basically this should not be difficult after we decide which tech approach to use and which endpoints are restricted.